File: python/1330.py
'''
    문제 이름 : 두수 비교하기
    문제 번호 : 1157
    문제 링크 : https://www.acmicpc.net/problem/1330
'''

a,b = map(int,input().split());

if a>b:
    print(">");
elif a<b :
    print("<");
else:
    print("==");

File: js/class_1/1330.js
const input = require("fs").readFileSync("dev/stdin").toString().split(" ");
let num1 = parseInt(input[0]);
let num2 = parseInt(input[1]);

if (num1 === num2) {
  console.log("==");
  return;
}

console.log(num1 < num2 ? "<" : ">");
